src/TLStacks1155.sol

```diff
@@ -477,6 +500,16 @@ contract TLStacks1155 is
         emit ProtocolFeeUpdated(newProtocolFeeReceiver, newProtocolFee);
     }
 
+    /// @notice Internal function to update the protocol fee split settings
+    /// @param newFreeMintFeeSplit The new fee split for free mints
+    /// @param newReferralFeeSplit The new fee split for referrals
+    function _setProtocolFeeSplits(uint256 newFreeMintFeeSplit, uint256 newReferralFeeSplit) internal {
+        freeMintFeeSplit = newFreeMintFeeSplit;
+        referralFeeSplit = newReferralFeeSplit;
```

Comment:
might want to limit the amount being set

Reply:
good catch